### DISCUSSION TITLE
使用QQ浏览器打开Waline Management System无法正常显示

### DISCUSSION THREAD

**Discussion by FrozenLemonTee** [walinejs/waline#162] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/64787592/109444802-0c3eb600-7a79-11eb-9ccd-d6ef85cd0ff3.png)
我原来用qq浏览器一直不能从自己博客打开UI界面，打开其他使用了waline的博客的UI界面也不行。
![109445074-b61e4280-7a79-11eb-89f2-7608fe6aadd0](https://user-images.githubusercontent.com/64787592/109445987-c1726d80-7a7b-11eb-9413-032f3e8f0073.png)
注册界面通过URL直接访问可以显示，但是尝试注册用户也是失败的。
![109445138-e534b400-7a79-11eb-9421-4a9096ac7d02](https://user-images.githubusercontent.com/64787592/109446093-00a0be80-7a7c-11eb-8d3e-f75d0661c942.png)
我以为是网络问题，后面使用火狐浏览器可以正常打开并显示，不知道这是不是浏览器兼容的问题？

**Comment by lizheming** [comment, marked as answer]
应该是，可以控制台打开看看有什么报错？

**Reply by FrozenLemonTee** [reply]
![image](https://user-images.githubusercontent.com/64787592/109447758-5aef4e80-7a7f-11eb-84a5-f6660882efa2.png)

**Reply by lizheming** [reply]
哦，globalThis 这个不兼容，我之后处理下，感谢反馈

**Reply by FrozenLemonTee** [reply]
还有一点，就是在博客评论系统注册以后，用QQ浏览器以及手机部分浏览器（UC不行、华为自带的可以）打开博客显示不出自己的头像（在其他人的博客注册也是一样），但是换浏览器（火狐、Chrome）可以，这是怎么回事

**Reply by lizheming** [reply]
提供下页面和控制台截图看看吧

**Reply by lizheming** [reply]
已邮件收到截图，看了下应该和浏览器没什么问题，图片地址都是相同的，可能是部分浏览器上有图片缓存的原因，可以尝试清楚下缓存试试。另外后台兼容的问题也已经修复处理，你再试试，感谢反馈。

**Reply by FrozenLemonTee** [reply]
已解决，感谢！

**Reply by dawnlyn** [reply]
@lizheming 刚刚edge也出现这个问题
版本：版本 92.0.902.62 (官方内部版本) (64 位)
edge Chromium版

没想到今天第一次用就出现这个问题....

**Reply by lizheming** [reply]
@Violet-VE 请提供下控制台报错截图吧

**Reply by dawnlyn** [reply]
@lizheming 不好意思，截图慢了点(～￣▽￣)～ 
![edge_Error](https://violet-ve.oss-cn-shanghai.aliyuncs.com/imgs/Temp/edge%E6%8A%A5%E9%94%99%E4%BF%A1%E6%81%AF.png)
顺便一提，QQ浏览器也报错了：
![QQ_Error](https://violet-ve.oss-cn-shanghai.aliyuncs.com/imgs/Temp/QQ%E6%B5%8F%E8%A7%88%E5%99%A8%E6%8A%A5%E9%94%99%E4%BF%A1%E6%81%AF.png)

**Reply by dawnlyn** [reply]
版本	Windows 11 企业版
版本	21H2
安装日期	‎2021/‎7/‎16
操作系统版本	22000.100
体验	Windows Feature Experience Pack 421.18901.0.3


处理器	Intel(R) Core(TM) i7-10700K CPU @ 3.80GHz   3.79 GHz
机带 RAM	16.0 GB (15.9 GB 可用)
系统类型	64 位操作系统, 基于 x64 的处理器

**Reply by dawnlyn** [reply]
刚刚试了Chrome，报错Failed to load resource: the server responded with a status of 500 ()

**Reply by dawnlyn** [reply]
不知道是不是哪里配置不对....希望能协助排查

**Reply by lizheming** [reply]
@Violet-VE 你好，你这个是没有配置数据库的环境变量。请按照新的教程 https://waline.js.org/guide/get-started.html#vercel-%E9%83%A8%E7%BD%B2-%E6%9C%8D%E5%8A%A1%E7%AB%AF 配置一下环境变量重新部署一下。
